src/shared/components/Modal/Modal.styles.ts

```diff
@@ -15,20 +15,19 @@ export const StyledDialog = styled(Dialog)`
     max-width: 100rem;
     height: ${({ height }) => height || 'auto'};
     overflow-y: unset;
-    padding: ${theme.spacing(2, 0)};
   }
 `;
 
 export const StyledDialogTitle = styled(DialogTitle)`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
   && {
-    padding: ${theme.spacing(2, 3.2, 3.4)};
+    padding: ${theme.spacing(3.2, 3.2, 0.8)};
   }
 `;
 
 export const StyledCloseButton = styled(StyledClearedButton)`
-  position: absolute;
-  top: 1.5rem;
-  right: 3.2rem;
```

Comment:
Thank you 😍 

For it to align even more pixel-perfectly, I believe it should also have an additional 4px margin (<img width="33" alt="Screenshot 2024-04-15 at 5 11 49 PM" src="https://github.com/ChildMindInstitute/mindlogger-admin/assets/1282772/0e2008ac-4658-4f75-a74c-93e0e8238e29">), which will also bump the title down a few pixels to better match Figma:
```tsx
  margin: 0.4rem;
```

Reply:
Those are those eagle eyes you've got there mate 🎉

Reply:
Lol… I have a bit of a reputation